docs/reference/tctl/batch/start.md

```diff
@@ -1,28 +1,26 @@
 ---
 id: start
 title: tctl batch start
+sidebar_label: start
```

Comment:
@djmagee assuming we go this route, we would make sure to specify what shows up in the sidebar.